src/components/Header.js

```diff
@@ -0,0 +1,22 @@
+import { useLocation, useHistory } from 'react-router-dom';
+import './Header.css';
+import { ReactComponent as ChevronLeftIcon } from './assets/icons/chevron-left.svg';
+
+const Header = () => {
+  const location = useLocation();
+  const history = useHistory();
+
+  return (
+    <header className="header">
+      {location.pathname === '/details'
+        ? <button type="button" aria-label="Go back" className="header-back-icon" onClick={() => history.goBack()}><ChevronLeftIcon /></button>
+        : <div />}
+      <div>
+        <h1 className="header-title">COVID Data Metircs App</h1>
+      </div>
+      <div />
+    </header>
```

Comment:
- [ ] The arrow is overlapping with the text from 320px to 370px. Please fix that.

![Screenshot from 2021-09-09 23-21-22](https://user-images.githubusercontent.com/22434660/132737353-a273e887-88b9-4a9a-a043-56cbd37c8466.png)

Reply:
Fixed